src/App.js

```diff
@@ -1,20 +1,441 @@
 import React from "react";
-import logo from "./logo.png";
+
 import "./App.css";
+import Header from "./components/Header";
+
+import TaskList from "./components/TaskList";
+import TaskListInProgress from "./components/TaskListInProgress";
+import TaskListInReview from "./components/TaskListInReview";
+import TaskListCompleted from "./components/TaskListCompleted";
 
 class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      tasks: [
+        {
+          id: 0,
+          title: "Reverse Engineer the Type-33 Needler",
+          task: "Coding a software to reverse engineer the Needler for Sacred Promissory",
+        },
+        // {
+        //   id: 1,
+        //   title: "Fixing bugs",
+        //   task: "Fixing coding bugs on the software for Machine Learning AI",
+        // },
```

Comment:
Let's remove all these comments. Those just waste space and do not add any value